ebuild-writing/eapi/text.xml

```diff
@@ -369,19 +369,29 @@ src_compile() {
 		</p>
 		<p>Example:</p>
 		<codesample lang="ebuild">
-EAPI=1
+EAPI=2
 ...
-src_install() {
-	emake DESTDIR="${D}" install || die "make install failed"
-	dodoc ChangeLog README
+src_test() {
+	if ! emake check ; then
+		eerror "Tests failed. Looking for files for you to add to your bug report..."
+		find "${S}" -type f -name '*.epicfail' -or -name '*.log' | while read a ; do
+			eerror "    ${a}"
+		done
```

Comment:
The while loop is executed in a subshell, therefore you don't have to declare `a` as a local variable. So the example is correct, but maybe this is a little too subtle?